mesh/settings.py

```diff
@@ -153,11 +155,42 @@
 
 STATIC_URL = 'static/'
 
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, 'meshapp/build/static')
+]
+
+STATIC_ROOT = os.path.join(BASE_DIR, 'meshapp/static')
+
 # Default primary key field type
 # https://docs.djangoproject.com/en/4.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
-STATICFILES_DIRS = [
-    os.path.join(BASE_DIR, 'meshapp/build/static')
-]
\ No newline at end of file
+# Djoser (User Management, Authentication, etc)
+# https://djoser.readthedocs.io/en/latest/introduction.html
+
+DJOSER = {
+    'LOGIN_FIELD':'email',
+    'USER_CREATE_PASSWORD_RETYPE': True,
+    'PASSWORD_CHANGED_EMAIL_CONFIRMATION': True,
+    'SEND_CONFIRMATION_EMAIL': True,
```

Comment:
I'm just curious about this is I have never used Djoser or seen much about it, but is it correct to believe that Djoser could handle email confirmation for accounts?

Reply:
Yes, most definitely